e2e/js/src/js-packaging.test.ts

```diff
@@ -0,0 +1,171 @@
+import {
```

Comment:
This was a rename from `js-bundling.test.ts` with additional tests, not sure why it wasn't marked as such.